snapcraft/storeapi/__init__.py

```diff
@@ -635,7 +635,7 @@ def snap_revisions(self, snap_id, series, arch):
             qs['series'] = series
         if arch:
             qs['arch'] = arch
-        url = 'snaps/' + snap_id + '/history'
+        url = 'snaps/' + snap_id + '/revisions'
```

Comment:
I got too eager.  Fixed.  This should explain why `revisions` and `list-revisions` was not working for you.